### PR TITLE
#166789989 Users Should Receive a JWT on Successful Registration

### PR DESCRIPTION
**Pull request by elemanhillary-zz**
#### What does this PR do?
- Add json web token features such as(generate token and check token)

#### Description of Task to be completed
- Ensures user receives json web token on successful registration

#### How should this be manually tested?
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-auth-jwt-166789989
npm install
npm start
```
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
[166789989](https://www.pivotaltracker.com/story/show/166789989)
#### Screenshots (if appropriate)
Imgur
#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by IsaiahRn — CHANGES_REQUESTED** [review]
1. Add a line on file `.editorConfig`
2. Capitalize every first letter on PR title

**Review by Hervera — CHANGES_REQUESTED** [review]
Tests are not passing [Check this error](https://travis-ci.com/andela/ah-92explorers-backend)

File: models/databaseUrl.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const pg = require("pg");
-const dotenv = require("dotenv");
+const pg = require('pg');
```
Comment (niyoceles) [inline models/databaseUrl.js]: I think this file `databaseUrl.js` It can be removed it doesn't have any value. because we are using sequelizer

**Review by niyoceles — CHANGES_REQUESTED** [review]

File: models/databaseUrl.js (hunk 1 above)
Comment (kodek-sleuth) [reply models/databaseUrl.js]: I think you should use es6

**Review by IsaiahRn — DISMISSED** [review]

**Review by Hervera — DISMISSED** [review]

**Review by IsaiahRn — DISMISSED** [review]
It is ready to be merged!

**Review by niyoceles — DISMISSED** [review]

**Review by Hervera — APPROVED** [review]

**Pull request closed without merging**